src/components/questionTypes/Free/creation/Content.js

```diff
@@ -0,0 +1,29 @@
+// @flow
```

Comment:
This component will be the same as for the other question types, we can put this in commons.